### PR TITLE
Conversations lead to merges blog post

### PR DESCRIPTION
**Pull request by syncrou**
Last week in ManageIQ - 10-17-2016

### CODE REVIEW

**Comment by hayesr** [comment]
LGTM 👍

**Comment by JPrause** [comment]
Awesome, thanks @syncrou! 
LGTM :+1: 

@Fryguy @jntullo @himdel can you review for this tomorrow's merge of this blog post.

File: site/_posts/2016-10-17-last-week-in-manageiq-conversations-lead-to-merges.html.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+### [CRUD for OpenStack Host Aggregates](https://github.com/ManageIQ/manageiq/pull/11580)
+
+With extreme attention to detail, [sseago](https://github.com/sseago) chatted 36 different times to hone and build
+API calls for creating, editing, deleting, and adding/removing hosts
+for OpenStack Host Aggregates.
+
+### [Chargeback for container images](https://github.com/ManageIQ/manageiq/pull/10677)
+
+[Ari](https://github.com/zeari) had some back and forth, and back and forth and... Whoila!
```
Comment (himdel) [inline site/_posts/2016-10-17-last-week-in-manageiq-conversations-lead-to-merges.html.md]: Not sure if it's supposed to be "Whoila" or just bad spelling of "Voilà"... ;)

**Comment by himdel** [comment]
I'd love to review but all I get is a #404 :trollface: 

But looks great, fun approach :)

**Comment by JPrause** [comment]
@himdel Nice! :) You had me there for a second.

**Comment by jntullo** [comment]
Love this different perspective @syncrou!! 👍 one suggestion because that sentence was confusing to me on first read (but it could just be me)

File: site/_posts/2016-10-17-last-week-in-manageiq-conversations-lead-to-merges.html.md (hunk 1 above)
Comment (syncrou) [inline site/_posts/2016-10-17-last-week-in-manageiq-conversations-lead-to-merges.html.md]: @himdel - Haha! - Nope - spelling fail on my part - I'll get that fixed. 😆

File: site/_posts/2016-10-17-last-week-in-manageiq-conversations-lead-to-merges.html.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+Although this technically feels like a new API feature, it is an improvement on the features
+exposed through our API.  [Tim](https://github.com/imtayadeway) spent a good amount of time 
+disscussing the changes needed over a 29 part conversation.
+
+## Fixed
+
+### [Allow to add a containers provider with a port other than 8443](https://github.com/ManageIQ/manageiq/pull/11391)
+
+14 separate discussions and now [Daniel Korn](https://github.com/dkorn) solved a problem where 
+when adding a containers provider, port 8443 was being used for the "default" role endpoint, even if a different port was specified.
```
Comment (syncrou) [inline site/_posts/2016-10-17-last-week-in-manageiq-conversations-lead-to-merges.html.md]: Thanks @jntullo - I'll get that change in.

**Comment by JPrause** [comment]
@hayesr looks like we're good to go. Can you please merge.

**Pull request merged**